stdlib/2and3/mmap.pyi

```diff
@@ -0,0 +1,81 @@
+# Stubs for mmap
+
+from typing import (
+    Optional, Sequence, Union,
+    Generic, TypeVar, overload
+)
+import sys
+
+
+_T = TypeVar('_T', str, bytes)
+
+# TODO already in PEP, have to get added to mypy
+from typing import Type
+from types import TracebackType
+_C = TypeVar('_C')
+class ContextManager(Generic[_C]):
```

Comment:
I know this is only temporary, but since it's something not actually exposed by mmap, do you want to add a leading `_`?